lib/core/src/server/preview/iframe-webpack.config.js

```diff
@@ -44,6 +71,16 @@ export default ({
       publicPath: '',
     },
     plugins: [
+      stories && stories.length
+        ? new VirtualModulePlugin({
+            [path.resolve(path.join(configDir, `generated-entry.js`))]: `
+              import { configure, addDecorator, addParameters } from '@storybook/react';
```

Comment:
Does this need to change to `@storybook/${framework}`??

Reply:
Framework is currently used in all the presets from `options.framework` 😉

Reply:
yes

Reply:
✅